content/blog/2024/11/kubewarden-1-19-release.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Kubewarden 1.19 release
+authors:
+  - José Guilherme Vanz
+date: 2024-11-28
+---
+
+We're excited to announce the release of Kubewarden v1.19! This release brings
+a host of improvements focused on minor bug fixes, adding tests, and developer
+tech debt improvements. 
+
+# Bug Fixes and Dependency Updates 
+
+As always, we've addressed bugs and updated dependencies to ensure a smooth and
+reliable experience. Notably, we've updated the dependencies for our major
+components. These updates contribute to the overall stability and security of
+the Kubewarden stack.
+
+# Fix to Policy-Server CI and SBOM file generation 
+
+In our previous release,
+[v1.18](https://www.kubewarden.io/blog/2024/11/kubewarden-1-18-release-slsa-level-3/),
+we achieved SLSA Lvl 3 compliance, a significant milestone in our commitment to
+supply chain security. However, we identified an issue where the SBOM (Software
+Bill of Materials) files for the policy server were being generated empty. This
+has been fixed in v1.19, ensuring accurate and complete SBOM generation.
+
+Beyond that, we also changed our CI to allow Kubewarden users to use the
+[slsactl](https://github.com/rancherlabs/slsactl) CLI tool to verify the
+container images. This tool can find and verify both SBOM and provenance files
+attached to the container images. However, it was failing to verify the
+policy-server container image due to the unexpected subject generated by the
+Github workflows. This is now fixed, and users can use slsactl to verify all
+our container images. Let’s take a look at an example of how to do this. To
+verify the Kubewarden policy-server container image, the following command can
+be used:
+
+```console
+slsactl verify ghcr.io/kubewarden/policy-server:v1.19.0
+```
+
+slsactl also allows users to download the files from the container image:
+
+```console
+slsactl download provenance ghcr.io/kubewarden/policy-server:v1.19.0
+slsactl download sbom ghcr.io/kubewarden/policy-server:v1.19.0
+```
+
+All the above commands can be used to verify the Kubewarden controller and
+audit scanner container images.
+
+# Removal of Old Post-Install Helm Chart Hooks
+
+We've cleaned up some legacy code by removing old post-install Helm chart hooks
+that were necessary for older releases. This streamlines the installation
```

Comment:
maybe it would be good to mention the supported upgrade path (update without jumping minor versions).

Reply:
Done